2018/solution4.py

```diff
@@ -0,0 +1,125 @@
+import sys
+import datetime
+from common import read_strings
+
+
+def guard_minute_optimizer_naive(guard_logs):
+    guard_map = parse_guard_logs(guard_logs)
```

Comment:
This looks like we're calling this a guard map because it's returning a dict. I haven't read the challenge (shame), but my first reaction is that this is a physical graph